current_samples_analysis.test.py

```diff
@@ -0,0 +1,8 @@
+import unittest
+import current_samples_analysis
+
+class TypewiseTest(unittest.TestCase):
```

Comment:
Good start 👍 
The test cases should be included with test set of single value array, multiple ranges within same input for ex. {3} , {3.3},
{1,2,3,5,6,7,8,10,}